CMSgov/mint-app
# Easi 2403 - Related content link and Help and Knowledge Center/Articles

# EASI-2403

## Changes and Description

- Created Help and Knowledge Center(HaKC) home
- Created `ArcticleCard` component for HaKC
- Created `RelatedArticles` component for HaKC
- Create `HelpBreadcrumb` component for handling dynamic breadcrumbs/close tab
- Task List link to related content that opens in new tab
- Consolidated/reused `Steps Involved` with the HaKC article

![Screen Shot 2022-10-14 at 10 13 08 AM](https://user-images.githubusercontent.com/95709965/195868307-ccf79e6b-c46b-40af-b2b0-3518a87c2bf3.png)

![Screen Shot 2022-10-14 at 10 13 24 AM](https://user-images.githubusercontent.com/95709965/195868384-e2340049-36fa-4880-9be3-e56c0e83fd9d.png)

![Screen Shot 2022-10-14 at 10 13 34 AM](https://user-images.githubusercontent.com/95709965/195868433-c5915e54-d3ca-4f66-85fd-e283cc85c683.png)

![Screen Shot 2022-10-14 at 10 13 53 AM](https://user-images.githubusercontent.com/95709965/195868500-a9c54f93-369c-4a52-af94-bed942aac1ed.png)

## How to test this change

- Click the `Help` tab on the navigation bar
- Verify two help article cards
- Click anywhere on `Overview for adding a model` card to enter article
- Verify the content, breadcrumbs, and related articles
- Navigate to task list and click the right nav link for `Overview for adding a model`
- Ensure this link opens in new tab and article breadcrumb becomes a link to close the tab

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
